cfgov/v1/util/util.py

```diff
@@ -13,4 +13,12 @@ def id_validator(id_string, search=re.compile(r'[^a-zA-Z0-9-_]').search):
 def to_camel_case(snake_str):
     snake_str = snake_str.capitalize()
     components = snake_str.split('_')
-    return components[0] + "".join(x.title() for x in components[1:])
\ No newline at end of file
+    return components[0] + "".join(x.title() for x in components[1:])
+
+class UniqueId(object):
+	index = 0
+
+	@classmethod
+	def get_unique_id(cls, prefix='', suffix=''):
+		cls.index = cls.index + 1
```

Comment:
So they have `uuid.uuid4()` but I wasn't sure if that was too long. 

https://docs.python.org/2/library/uuid.html